game/scripts/npc/abilities/weaver_time_lapse.txt

```diff
@@ -16,10 +16,10 @@
 
     "HasScepterUpgrade"                                   "1"
 
-    "AbilityCastPoint"                                    "0.3"
+    "AbilityCastPoint"                                    "0.2" //OAA
```

Comment:
Dont buff the cast point on time laspe. It should be left unchanged.

Reply:
^ this hero is really underrated i feel as a utility support and this is a unneeded buff.

Reply:
My idea was to put 0 cast point on weaver time lapse when without aghs. But I couldn't do that without changing cast point with aghs, Compromise was 0.2. 
@lovesmuffins I know that weaver support is strong so I nerfed the aghs cd to compensate. It was more craizy when octarine scaled.

Reply:
I agree with what loves says, I think weaver is a little underrated. I really am not a fan of this cast point reduction. All it does is reduce counterplay in an unfair way. 

Weavers main issue is that its hard to fit him on a team. He can't roam and he doesn't farm fast. He also isn't an S+ tier late game carry.

Reply:
> My idea was to put 0 cast point on weaver time lapse when without aghs. But I couldn't do that without changing cast point with aghs, Compromise was 0.2.
> @lovesmuffins I know that weaver support is strong so I nerfed the aghs cd to compensate. It was more craizy when octarine scaled.

cast point buff is silly in my opinion. I would appreciate perhaps a W dmg buff early levels or perhaps increase hits to destroy on his Q instead.

Reply:
Cast point is improved now only at OAA levels.

Reply:
still doesn't really feel good in my stomach but I assume it won't have a huge impact and any weaver buff is good atm. So I'm fine with this change I guess.